webpack.config.js

```diff
@@ -396,7 +396,7 @@ if (WEBPACK_BUNDLE !== "production") {
   // with BETTER_SOURCE_MAPS we switch to sourcemaps that work with breakpoints and makes stacktraces readable
   config.devtool = process.env.BETTER_SOURCE_MAPS
     ? "eval-source-map"
-    : "cheap-module-source-map";
+    : "eval-cheap-module-source-map";
```

Comment:
I don't always use source maps because of unreliable breakpoints in browser dev tools.

This is how the bundle without source maps looks like in `master`: [image](https://github.com/user-attachments/assets/09567664-7464-4314-a63e-467ad8a18258) - I often debug it this way

This is how bundle without source maps looks like with this change: [image](https://github.com/user-attachments/assets/62d6398f-1068-464f-b9c3-9b03c509c347) - Undebuggable

Can we keep it as is? :pray:

Reply:
wow, you're the first person I see who doesn't use sourcemaps 😄 maybe instead we can have a flag to start fe server without sourcemaps but with more or less readable code